demos/mkfile

```diff
@@ -0,0 +1,31 @@
+APE=/sys/src/ape
```

Comment:
please try to integrate in the "new" build environment - there's a common directory and if possible the Makefile includes defined there should be used - otherwise it should go into the port directory.

Reply:
This has been done and also for the upstream PR